cli/tsc/99_main_compiler.js

```diff
@@ -725,10 +725,6 @@ delete Object.prototype.__proto__;
           if (item) {
             isCjsCache.add(item);
             const [resolvedFileName, extension] = item;
-            if (resolvedFileName.startsWith("node:")) {
-              // probably means the user doesn't have @types/node, so resolve to undefined
-              return undefined;
-            }
```

Comment:
And this one?

Reply:
Moved to rust in `op_resolve()` implementations.